TypeCobol/LanguageServices/Editor/ILanguageServer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler;
+using TypeCobol.Compiler.Concurrency;
+using TypeCobol.Compiler.Scanner;
+using TypeCobol.Compiler.Text;
+
+namespace TypeCobol.LanguageServices.Editor
```

Comment:
Now that we have a generic listener in CompilationDocument can you move this class to the TypeCobol.LanguageServer project ?